### DISCUSSION TITLE
Request for an Option to Use Chinese in Copilot Conversations

### DISCUSSION THREAD

**Discussion by tylike** [community/community#61889] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I hope Copilot can always converse with me in Chinese, but to achieve this, I have to declare every time I start a conversation: "Please use Chinese to talk to me, and then state my real question." It seems that Copilot doesn't remember our previous conversation even if I haven't pressed the "Clear History" button. Is it possible to add an option so that I don't have to make this declaration every time? It's very troublesome.

Chinese Version:
我希望copliot始终用中文和我对话，但要实现这点，我必须每次对话时都需要声明：“请使用中文和我对话，后面才是我的真实问题”，它似乎不记得我们以前的对话内容。即使我没有按下“清除历史记录按钮”，是否能够增加一个选项，使我不必每次都做这个声明？这很麻烦.

**Comment by BaraaBasim** [comment]
In order to change the language of GitHub Copilot, you need to modify the language settings of your IDE or code editor.
